frontend/src/pages/HomePage.module.scss

```diff
@@ -1,12 +1,17 @@
 .Home {
   padding: 7vh 7vw;
   height: 100vh;
-  background-image: url('../assets/home.png');
   background-position: center;
   background-repeat: no-repeat;
   background-size: cover;
+  background-color: #f6f6f6;
 }
 
 .search {
   margin-top: 50px;
 }
+
+.homepageDescription {
+  padding-left: 7%;
```

Comment:
We generally want to shy away from magic percentage numbers like these because they might behave weirdly on mobile. Instead, maybe consider using `px` or `em`.